docs/samples/fabric/notebook_execution.md

```diff
@@ -0,0 +1,80 @@
+## Running Presidio and Spark in a Fabric Notebook
```

Comment:
Can't this whole documentation piece, or at least from bullet no. 4, be incorporated inside the notebook, not having to hop between the two?